modules/serverless-installing-cli-linux.adoc

```diff
@@ -0,0 +1,36 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="installing-cli-linux_{context}"]
+= Installing the `kn` CLI for Linux
+
+For Linux distributions, you can download the CLI directly as a `tar.gz` archive. 
+
+
+.Procedure
+
+. Download the link:https://mirror.openshift.com/pub/openshift-v4/clients/serverless/latest[CLI].
```

Comment:
Just want to double check that this link should work (404s now) at GA?

Reply:
Work is being done to bring this link up.

Reply:
yeah. That is where it will be published.

Reply:
Thanks for confirming!